Source/Scene/SceneTransitioner.js

```diff
@@ -870,6 +870,12 @@ define([
                 camera.frustum = camera3D.frustum.clone();
             }
 
+            var frustum = camera.frustum;
+            if (scene._logDepthBuffer && !(frustum instanceof OrthographicFrustum || frustum instanceof OrthographicOffCenterFrustum)) {
+                frustum.near = 0.1;
```

Comment:
Same comment here and below.

Reply:
Ah, there were some hidden comments.

Reply:
This has the same problem as in master. When the scene mode switches, we reset to the default near far distances in the frustum constructor. The old ones aren't saved between scene mode changes (should happen, separate PR or open an issue). This just sets to better defaults when log depth is enabled.